### PR TITLE
Added source URL for vim cheat sheet

### LINKED ISSUE #1278: Title of vim cheat sheet Instant Answer
Look at any cheat sheet - [gimp](https://duckduckgo.com/?q=gimp+cheat+sheet&ia=cheatsheet), [blender](https://duckduckgo.com/?q=blender+cheat+sheet&ia=cheatsheet), [harry potter spells](https://duckduckgo.com/?q=harry+potter+spells+cheat+sheet&ia=cheatsheet), [gnome keyboard](https://duckduckgo.com/?q=gnome+keyboard+cheat+sheet&ia=cheatsheet), and all others.
The title of the result page will be - cheat_sheet_name at DuckDuckGo
Example [1] : Title of gimp cheat sheet is - gimp cheat sheet at DuckDuckGo
Example [2] : Title of blender cheat sheet is - blender cheat sheet at DuckDuckGo

But, for [vim cheat sheet](https://duckduckgo.com/?q=vim+cheat+sheet&ia=cheatsheet) the title is - vim cheat sheet (Software) at DuckDuckGo 

The word (Software) is appended. :frowning:

### PR DESCRIPTION
**Pull request by sagarhani**
This will fix #1278.

### CODE REVIEW

File: share/goodie/cheat_sheets/vim.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "description": "Text Editor",
     "metadata": {
         "sourceName": "VimCheatSheet",
-        "sourceUrl": "https://duckduckgo.com"
+        "sourceUrl": "vim.rtorr.com"
```
Comment (javathunderman) [inline share/goodie/cheat_sheets/vim.json]: Nitpicking here- can you put a "http://" in front of the subdomain (in keeping with other CheatSheets")?

**Comment by javathunderman** [comment]
Hey @sagarhani, I left a line note for you. Besides that it LGTM. :+1: 

cc @moollaza

File: share/goodie/cheat_sheets/vim.json (hunk 1 above)
Comment (mintsoft) [inline share/goodie/cheat_sheets/vim.json]: Or https if available!

Comment (javathunderman) [inline share/goodie/cheat_sheets/vim.json]: https>http :smile:

**Comment by moollaza** [comment]
LGTM :+1: 

@sagarhani thanks!

**Pull request merged**